dot/core/service.go

```diff
@@ -587,33 +610,34 @@ func (s *Service) ProcessBlockResponseMessage(msg network.Message) error {
 				Body:   body,
 			}
 
-			enc, err := block.Encode()
-			if err != nil {
-				return err
-			}
+			// TODO: why doesn't execute block work with block we built?
```

Comment:
was this happening before? should we open an issue to address or are you planning to address here?

Reply:
I don't think it was every tested :/ I'll open an issue to address it

Reply:
#681